package.json

```diff
@@ -32,6 +32,7 @@
     "build": "npm-run-all build:*",
     "build:ga": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js",
     "build:manifest": "mustache package.json manifest.json.tmpl > addon/manifest.json",
+    "build:buildSettings": "node -e 'console.log(JSON.stringify(process.env))' | mustache - buildSettings.js.tmpl > addon/build/buildSettings.js",
```

Comment:
This seems so obtuse, but eh...

Reply:
Yeah, Webpack would probably make this a lot cleaner than inline Node to stdout mustache templates, but Webpack would be a pretty big hammer for something this small.